src/containers/stats/courttype/MonthlyParticipantsByCourtType.js

```diff
@@ -66,8 +63,20 @@ const MonthlyParticipantsByCourtTypeList = ({ actions, monthlyParticipantsByCour
   const currentYearOption :Object = YEARS_OPTIONS.find((obj) => obj.value === today.year);
   const [year, setYear] = useState(currentYearOption);
 
-  const getNewData = () => {
-    actions.getMonthlyParticipantsByCourtType({ month: month.value, year: year.value });
+  const onChangeMonth = (newMonth :Object) => {
+    setMonth(newMonth);
+    actions.getMonthlyParticipantsByCourtType({
+      month: newMonth.value,
+      year: year.value,
+    });
+  };
+
+  const onChangeYear = (newYear :Object) => {
+    setYear(newYear);
+    actions.getMonthlyParticipantsByCourtType({
+      month: month.value,
+      year: newYear.value,
+    });
   };
```

Comment:
1) You could use the same handler and leverage the event.target.name value to distinguish the Select elements
2) It seems that `actions.getMonthlyParticipantsByCourtType` is being called on every select. I think it makes sense to only invoke the action when the year and month value changes. (consider wrapping in a useEffect)

Reply:
done! good suggestion, i did this for all of the graphs that have timeframe selects.